src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1110,6 +1110,29 @@ private StringBuilder getRowsOfPossibleRecipientsInCsvFormat(
         }
         return sessionsToSendEmailsFor;
     }
+    
+    /**
+     * Criteria: must be published, publishEmail must be enabled and
+     * resultsVisibleTime must be custom.
+     * 
+     * @return returns a list of sessions that require automated emails to be
+     *         sent as they are published
+     */
+    public List<FeedbackSessionAttributes> getFeedbackSessionsWhichNeedAutomatedUnpublishedEmailsToBeSent() {
+        List<FeedbackSessionAttributes> sessions =
+                fsDb.getFeedbackSessionsWithUnsentUnpublishedEmail();
+        List<FeedbackSessionAttributes> sessionsToSendEmailsFor =
+                new ArrayList<FeedbackSessionAttributes>();
+
+        for (FeedbackSessionAttributes session : sessions) {
+            // automated emails are required only for custom unpublish times
+            if (!session.isPublished() && session.isPublishedEmailEnabled()
```

Comment:
I got that now. I misunderstood that.. actually I thought we are not using cron jobs for publishing sessions. 
Just one doubt, Do we have the functionality for unpublishing result in an automatic way, like setting time for unpublishing the result ? 
I messed up everything, it seems much trivial now. will update soon.